shared/types/submission.ts

```diff
@@ -73,6 +74,28 @@ export type StorageModeSubmissionBase = z.infer<
   typeof StorageModeSubmissionBase
 >
 
+/**
+ * Multirespondent submission typings as stored in the database.
+ */
+
+export const MultirespondentSubmissionBase = SubmissionBase.extend({
```

Comment:
Why not extend `StorageModeSubmissionBase` with the additional fields instead since we will be going for feature parity in subsequent phases anyway?

Reply:
I don't think we can have feature parity (or that I know of) for things such as myinfo, payments?